File: modules/day-04-reliable/r/starter.R
pass_rate <- function(scores, passing_score = 60) {
  passed <- scores > passing_score
  sum(passed) / length(scores) * 100
}

scores_a <- c(55, 60, 62, 90)
scores_b <- c()

print(pass_rate(scores_a))
print(pass_rate(scores_b))

File: modules/day-04-reliable/python/starter.py
def pass_rate(scores, passing_score=60):
    passed = [score > passing_score for score in scores]
    return sum(passed) / len(scores) * 100


scores_a = [55, 60, 62, 90]
scores_b = []

print(pass_rate(scores_a))
print(pass_rate(scores_b))
